frostsnapp/README.md

```diff
@@ -0,0 +1,33 @@
+# frostsnapp
+
+This repository serves as a template for Flutter projects calling into native Rust
+libraries via `flutter_rust_bridge`.
+
+## Install
+
+To begin, ensure that you have a working installation of the following items:
+- [Flutter SDK](https://docs.flutter.dev/get-started/install)
+- [Rust language](https://rustup.rs/)
+- `flutter_rust_bridge_codegen` [cargo package](https://cjycode.com/flutter_rust_bridge/integrate/deps.html#build-time-dependencies)
+- Appropriate [Rust targets](https://rust-lang.github.io/rustup/cross-compilation.html) for cross-compiling to your device
+- For Android targets:
+    - Install [cargo-ndk](https://github.com/bbqsrc/cargo-ndk#installing)
+    - Install [Android NDK 22](https://github.com/android/ndk/wiki/Unsupported-Downloads#r22b), then put its path in one of the `gradle.properties`, e.g.:
+
+```
+echo "ANDROID_NDK=.." >> ~/.gradle/gradle.properties
```

Comment:
Might also need to use `flutter config --android-sdk  <ANDROID_NDK_DIR>`, or reopen terminal

Reply:
Hmmm I wonder why that would help. grade.properties is used by gradle not by your shell unless there's something I don't understand.

Reply:
I think it came up as a suggestion at some point during the install process. Maybe ignore for now